Sources/ForageSDK/Foundation/Telemetry/ForageLogger.swift

```diff
@@ -1,11 +1,17 @@
 //
-//  File.swift
+//  ForageLogger.swift
 //
 //
 //  Created by Danilo Joksimovic on 2023-07-26.
 //
 
-import Datadog
+// NOTE: This logger is built upon a "stripped" version of the Datadog package v2.1.0, found in the Sources/ForageSDK/Vendor directory.
+// "Stripped" refers to the inclusion of only the DatadogCore and DatadogLogs modules, which are essential for emitting logs as described at https://docs.datadoghq.com/logs/log_collection/ios/.
+// Initially, we decided against forking the Datadog package due to the complexity of resolving name clashes and other integration challenges.
```

Comment:
Since our packages are public, can we maybe put these into a notion document and link the document here? That way, those outside the organization can't see this info.